test-suite/Makefile

```diff
@@ -38,9 +38,9 @@ include ../Makefile.common
 BIN := $(shell cd ..; pwd)/bin/
 COQFLAGS?=
 
+COQLIB?=
```

Comment:
This is incorrect I think; welcome to the world of make where environment variables and make variables share the same namespace; so IIUC, you are unconditionally destroying the `COQLIB` envar here.

Please remove.

Reply:
`?=` shouldn't override environment variables

Reply:
Yet it does?

Reply:
Why do you think it does? I just tested and it didn't.

Reply:
Or else why it makes the build fail?

Reply:
My question is more general, why `F=foo make`  and `make F=foo` are not the same? I learned that once, but I forgot. [I am indeed wrong about `Foo ?=` overriding envars]

Reply:
in `make V=value` `$V` will be value unless it's set with a special keyword in the makefile. We never use the keyword so `make V=value` always forces `$V` to be `value`.
Values from the environment are overriden by any assignation except ?=